sql/001-create-table-and-load-data.sql

```diff
@@ -0,0 +1,11 @@
+DROP TABLE IF EXISTS names;
+
+CREATE TABLE names (
+  id int unsigned AUTO_INCREMENT,
+  name VARCHAR(20) NOT NULL,
+  PRIMARY KEY(id)
+  );
```

Comment:
些細な点ですが、フォーマットが気になります。
```suggestion
CREATE TABLE names (
  id int unsigned AUTO_INCREMENT,
  name VARCHAR(20) NOT NULL,
  PRIMARY KEY(id)
);
```

Reply:
ご指摘のスペースを修正しました。
![image](https://user-images.githubusercontent.com/113277395/201453688-d86c6b7f-0b2a-483b-b413-57fac0ed52e4.png)